packages/python-packages/apiview-stub-generator/tests/class_parsing_test.py

```diff
@@ -65,231 +81,388 @@ def test_class_with_decorators(self):
             "self, ",
             "id, ",
             "*args, ",
-            "**kwargs",                        
+            "**kwargs",
             ")",
         ]
         _check_all(actuals, expected, obj)
 
+        # last blank line is context end for class
+        assert len(tokens[-1]['Children'][-1]['Tokens']) == 0
+        assert tokens[-1]['Children'][-1]['IsContextEndLine'] == True
```

Comment:
isContextEndLine tests - may be better to go through all of them and count, rather than just checking a specific one.